src/components/StyledText.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, HTMLAttributes } from "react";
+import { css, StyleSheet } from "aphrodite";
+import { colors, typography } from "styles";
+
+interface Props {
+  error?: boolean;
+  style?: HTMLAttributes<HTMLParagraphElement>;
+}
+
+const StyledText: FC<Props> = ({ children, error, style }) => {
+  return (
+    <p
+      className={css(
+        styles.text,
+        style,
+        typography.body1,
+        error && styles.errorText
+      )}
+    >
+      {children}
+    </p>
+  );
+};
+
+const styles = StyleSheet.create({
+  text: {
+    fontFamily: "Roboto",
```

Comment:
So this couldn't be implemented via `body1`, yes? If yes import this font as const from typography

Reply:
It could be, and it is. It was in two places, and it works in any of them.